src/OrbitGl/App.cpp

```diff
@@ -2564,3 +2569,32 @@ void OrbitApp::CaptureMetricProcessApiTimer(const orbit_client_protos::TimerInfo
       UNREACHABLE();
   }
 }
+
+void OrbitApp::SaveCaptureUserInfo() {
+  CHECK(std::this_thread::get_id() == main_thread_id_);
+  if (!HasCaptureData()) return;
+  if (IsCapturing()) {
+    // We are going to save it at the end of capture anyways.
+    return;
+  }
+
+  const auto& file_path = GetCaptureData().file_path();
+  if (!file_path.has_value()) {
+    LOG("Warning: capture is not backed by a file, skipping the save.");
```

Comment:
Is this expected or not? If not I would just use `ERROR`. If yes, I would just drop the "Warning".